jerry-core/ecma/base/ecma-globals.h

```diff
@@ -79,6 +79,7 @@ typedef enum
   ECMA_SIMPLE_VALUE_UNDEFINED, /**< undefined value */
   ECMA_SIMPLE_VALUE_NULL, /**< null value */
   ECMA_SIMPLE_VALUE_NOT_FOUND, /**< a special value returned by ecma_op_object_find */
+  ECMA_SIMPLE_VALUE_NOT_FOUND_AND_STOP, /** not found and not search the proto chain */
```

Comment:
I understand what it does, the question is: do we really need it? What about simply return with undefined virtual data?